andela/colossus-backend
# #167733837 Create An Endpoint To Edit Requests

#### What does this PR do?
Creates an endpoint for a user to edit the details of a previously created request
#### Description of Task to be completed?

- Create an endpoint to edit requests

- Create an endpoint to delete requests

#### How should this be manually tested?
On Postman, make a PATCH request to {url}/api/v1/request/{requesId}
the req.body should contain the following fields: 
{
  "passportName": "Iyara Ferguson",
  "reason": "Pilgrimage",
  "type": "one-way",
  "from": ["Dubai"],
  "to": ["Sri-Lanka"],
  "departureDate": ["2018-03-29T13:34:00.000"],
  "accommodation": ["Four Points"],
  "id": [1]
}
To delete a trip, make a DELETE request to {url}/api/v1/trip/{tripId}
N.B: For both cases, a token has to be present in the headers
#### Any background context you want to provide?
req.body.id represents the id of the trip to be edited
#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/167733837](https://www.pivotaltracker.com/story/show/167733837)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
